scripts/import/import_cosmic.pl

```diff
@@ -308,10 +311,27 @@ sub get_source_id {
   return $source_id[0];
 }
 
+sub get_attrib_id {
+  my ($type, $value) = @_;
```

Comment:
You could add a comment saying which values are accepted for $type and $value. For $type you use 'evidence' and 'phenotype_type' but are more values accepted?

Reply:
The method can retrieve any `type, value` pairs based on the attrib and attrib_type tables. In this script I'm only using `evidence` and `phenotype_type` types.